src/popovers/_base.scss

```diff
@@ -103,7 +103,7 @@
     transform: translateY(-50%);
 
     &:after {
-      @include arrow('right', background, 15px, $shadow: true);
+      @include arrow('right', background, 15px, $shadow: true, $shadow-opacity: 0.05);
```

Comment:
Given that we are repeating values, I think it would be best to create a `config.scss` for this component and move some of the values into variables. Namely, color (background), arrow-size (15px), and shadow (0.05)